src/lib/open/index.ts

```diff
@@ -0,0 +1,62 @@
+const { Android, Ios } = require("uri-scheme")
+const dashify = require("dashify")
+const template = require("colon-template")
+const open = require("open")
+
+import config from "./config"
+interface OpenPageProps {
+  page: string
+  platform: string | undefined
+  environment: string | undefined
+  customVariables: object
+}
+
+const openUri = (uri: string, platform: string) => {
+  switch (platform) {
+    case "android":
+      Android.openAsync({ uri })
+      break
+    case "ios":
+      Ios.openAsync({ uri })
+      break
+    case "web":
+      open(uri)
```

Comment:
oclif has a built in utility for opening URLs in the browser: https://github.com/oclif/cli-ux#cliopen

Might be worth using so we can drop a dependency?

Reply:
👍